lib/vagrant/disksize/actions.rb

```diff
@@ -80,6 +80,14 @@ def attach_disk(driver, disk)
           device = parts[2]
           driver.execute('storageattach', @machine.id, '--storagectl', controller, '--port', port, '--device', device, '--type', 'hdd',  '--medium', disk[:file])
         end
+				
+        def remove_disk(driver, disk)
+          parts = disk[:name].split('-')
```

Comment:
We can remove lines 85 through 88 in remove_disk() since those variables are not used.